README.md

```diff
@@ -0,0 +1,147 @@
+<a name="readme-top"></a>
+
+
+<div align="center">
+
+  <h3><b>CATALOGUE OF THINGS</b></h3>
+   
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [OOP School Library] <a name="about-project"></a>
+
+> Catalogue of things is ruby project used record books, Music Album and games in the library. The library user is have multiple selection of choosing which activity to perfom at the library.
+Depending on wether he want either music, book, or game... he gets different selection on listing all the available items in the store, and few choices are given to add his favorite to the store. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<ul>
+  <li><a href="https://www.ruby-lang.org/en/news/2022/11/24/ruby-3-1-3-released/">Ruby 3.1</a></li>
+</ul>
+
+
+<!-- GETTING STARTED -->
+
+## Getting Started 
+
+> To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+  - <a href="https://www.ruby-lang.org/en/news/2022/11/24/ruby-3-1-3-released/">Ruby 3.1</a>
+  - Rubocop _(for local automated tests)_
+
+### Setup
+
+1. Clone this project on your local machine using the following command on your Git Bash: `https://github.com/Irirwanirira/Catalog-Ruby-project.git`
+
+2. Browse to project directory `cd Catalog-Ruby-project`
+
+3. Run `bundle install` to install the dependencies
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Sajeel Zafar**
+
+- GitHub: [@Sajeelzafar](https://github.com/Sajeelzafar)
+- Twitter: [@SajeelZafar6](https://twitter.com/SajeelZafar6)
+- LinkedIn: [Sajeel Zafar](https://www.linkedin.com/in/sajeelzafar/)
+
+👤 **Joseph Irirwanirira**
+
+- GitHub: [@githubhandle](https://github.com/Irirwanirira)
+- Twitter: [@twitterhandle](https://twitter.com/iri_joseph)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/Joseph Irirwanirira)
+
+👤 **Abdulai Samuel Moiwa**
+
+- GitHub: [@githubhandle](https://github.com/samuelmoiwa)
+- Twitter: [@twitterhandle](https://twitter.com/samuelmoiwa)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ing-abdulai-samuel-moiwa-726340142/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+>   Live demo video
+
+- [Live Demo Video](https://drive.google.com/file/d/1NF7t23iaPeGlTer4DE3FD5_X8rpD5Ic0/view?usp=share_link)
```

Comment:
[REQUIRED] When I click the link to your video it requires me to request access. Please make your video generally accessible with tools like loom

![image](https://user-images.githubusercontent.com/62938735/213695701-c0ff5efe-69ee-4cca-a6f6-9152a35bbe89.png)

Reply:
@fobadara we enabled the video share, Thank you.